backend/src/schema/types/type/NOTIFIED.gql

```diff
@@ -0,0 +1,28 @@
+type NOTIFIED {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:34:14Z" title="Friday, August 30th 2019, 2:34:14 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,28 @@
+type NOTIFIED {
```


why is this type all caps? I thought you were critcal of the `EMOTED` type??
we don't have consistent casing

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T13:12:57Z" title="Friday, August 30th 2019, 3:12:57 pm +02:00">Aug 30, 2019</time>_

---


Because it is a relation dear …

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-30T13:52:14Z" title="Friday, August 30th 2019, 3:52:14 pm +02:00">Aug 30, 2019</time>_

---


Yes, @ulfgebhardt added the convention of writing the graphql type in all caps if it gets mapped to an edge, not a node, in neo4j. I was complaining about it before I learned the reason for it.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T14:02:07Z" title="Friday, August 30th 2019, 4:02:07 pm +02:00">Aug 30, 2019</time>_

---


yea, thanks for clarifying @Tirokk, that makes sense